maintainers/maintainer-list.nix

```diff
@@ -97,6 +97,12 @@
     github = "1000teslas";
     githubId = 47207223;
   };
+  _2gn = {
```

Comment:
This file should be in a separate commit, with the title

`maintainers: add myself`

Reply:
Also, put `pngloss: init at unstable-2020-11-25` on the other commit (removing everything else).

Reply:
> This file should be in a separate commit, with the title
> 
> `maintainers: add myself`

My guess is they misunderstood my squash commits, because that commit used to exist

Reply:
I've splitted squashed commit into "maintainer: add myself" and "pngloss: init at unstable-2020-11-25". Was it correct?

Reply:
Yes. The idea is two commits, one for the new maintainer and other for the package itself.

Reply:
Thank you! I've learned a lot from this pr.